plugins/TodayAtMun/today.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime, timedelta
+from json import load
+from pathlib import Path
+from typing import Dict
+
+
+class Today:
+    """
+    A class used to go find significant days on Mun Calendar
+
+    ...
+
+    Attributes:
+    ------------
+    None
+
+    Methods
+    ----------
+    set_current_day() : None
```

Comment:
if you move these textual descriptions of the methods (i.e. `sets the current date at the moment called`) to the doc strings of the methods themselves, you can make this large doc string at the top much smaller (since it should only really describe the class as a whole, and document the methods below instead